src/tequila/quantumchemistry/chemistry_tools.py

```diff
@@ -50,16 +50,132 @@ def __init__(self, generator, p0, transformation, indices=None, *args, **kwargs)
         self._name = "FermionicExcitation"
         self.transformation = transformation
         self.indices = indices
-
+        if isinstance(indices,tuple) and not hasattr(indices[0],"__len__"):
```

Comment:
indices can in principle come in as a list and not a tuple.

e.g.
indices=(01,2,3) --> [(0,2),(1,3)] 
this will work

but 
indices = [0,1,2,3] will not work

since indices will in all-cases have an access operator defined, you can just skip the `isinstance(indices,tuple)`  part